system_tests/language.py

```diff
@@ -76,15 +76,15 @@ def _check_analyze_entities_result(self, entities):
         self.assertGreater(entity1.salience, 0.0)
         # Other mentions may occur, e.g. "painter".
         self.assertIn(entity1.name, entity1.mentions)
-        self.assertEqual(entity1.wikipedia_url,
+        self.assertEqual(entity1.metadata['wikipedia_url'],
```

Comment:
No actually, that change is already in, but the system tests did not reflect it (and fail on master).